requirements.txt

```diff
@@ -0,0 +1,7 @@
+bcrypt==4.1.2
+cffi==1.16.0
+couchbase==4.2.0
+cryptography==42.0.5
+paramiko==3.4.0
+pycparser==2.22
```

Comment:
Where do we use this lib `pycparser` ?

Reply:
From a quick search I can see that paramiko and cryptography internally use pycparser. These dependencies with the virtual environment.

Reply:
Changes made in latest commits